File: c/01-estrutura-sequencial/soma.c
/*Problema "soma"
Fazer um programa para ler dois valores inteiros X e Y, e depois mostrar na tela o valor da soma destes
números.*/

#include <stdio.h> 
int main () {
    int x, y, sum; 
    
    printf("DIgite o valor de X: ");
    scanf("%d", &x);
    printf("DIgite o valor de Y: ");
    scanf("%d", &y);

    sum = x + y; 

    printf("SOMA = %d\n", sum); 

    return 0;
}

File: java/01-estrutura-sequencial/Soma.java
/*Problema "soma"
Fazer um programa para ler dois valores inteiros X e Y, e depois mostrar na tela o valor da soma destes
números.*/

import java.util.Locale;
import java.util.Scanner;

public class Soma {

	public static void main(String[] args) {
		Locale.setDefault(Locale.US);
		Scanner sc = new Scanner(System.in);
		
		int x, y, sum;
		
		System.out.print("Digite o valor de X: ");
		x = sc.nextInt();
		System.out.print("Digite o valor de Y: ");
		y = sc.nextInt();
		
		sum = x + y;
		
		System.out.print("SOMA = " + sum);
		
		sc.close();
	}

}
